src/components/camera-tool.js

```diff
@@ -295,14 +297,14 @@ AFRAME.registerComponent("camera-tool", {
       this.durationLabel.setAttribute("text", "value", `${this.data.captureDuration}`);
     }
 
-    this.durationLabel.object3D.visible = hasDuration && !this.data.isSnapping && !isFrozen;
-    this.recordIcon.object3D.visible = !hasDuration && !isPhoto && !isFrozen;
-    this.recordAlphaIcon.object3D.visible = hasDuration && !isPhoto && !isFrozen;
-    this.snapIcon.object3D.visible = isPhoto && !isFrozen;
+    this.durationLabel.object3D.visible = hasDuration && !this.data.isSnapping && !isFrozen && allowVideo;
+    this.recordIcon.object3D.visible = !hasDuration && !isPhoto && !isFrozen && allowVideo;
+    this.recordAlphaIcon.object3D.visible = hasDuration && !isPhoto && !isFrozen && allowVideo;
     this.snapButton.object3D.visible = !this.data.isSnapping && !isFrozen;
+    this.recordButton.object3D.visible = !this.data.isSnapping && !isFrozen && allowVideo;
     this.cancelButton.object3D.visible = this.data.isSnapping && !isFrozen;
     this.prevDurationButton.object3D.visible = this.nextDurationButton.object3D.visible =
-      !this.data.isSnapping && allowVideo && !isFrozen;
+      !this.data.isSnapping && allowVideo && !isFrozen && allowVideo;
 
```

Comment:
Can't see countdown label from behind the camera, so it seems like nothing happened after clicking the photo/video buttons.

Reply:
(Problem is not specifically at this line)

Reply:
fixed